fhir-install/src/main/assembly/distribution.xml

```diff
@@ -68,17 +73,6 @@
                 <include>${project.groupId}:fhir-config</include>
             </includes>
         </dependencySet>
-        <dependencySet>
-            <useProjectArtifact>false</useProjectArtifact>
-            <outputDirectory>artifacts/servers/fhir-server/userlib</outputDirectory>
-            <directoryMode>0755</directoryMode>
-            <fileMode>0755</fileMode>
-            <includes>
-                <include>${project.groupId}:fhir-ig-*</include>
-                <!-- TODO: move operations to userlib -->
-                <!-- <include>${project.groupId}:fhir-operation-*</include> -->
-            </includes>
-        </dependencySet>
```

Comment:
I'd rather leave this file alone.  Did the approach I shared not work? if so, the profile order makes a difference.

Reply:
It did, but it would show a warning like this when not including the IGs:
[WARNING] The following patterns were never triggered in this artifact inclusion filter:
o  'com.ibm.fhir:fhir-ig-*'
Maybe that is ok, but I thought it would be nice to a warnings like that show up by default.

Reply:
I think the warning is OK. Maybe a note in the distribution.xml `<!-- included as there are some transitive dependencies picked up during integration test builds. -->`

Reply:
Done.